src/DataViews/LiveFunctionsDataViewTest.cpp

```diff
@@ -213,10 +218,6 @@ std::unique_ptr<CaptureData> GenerateTestCaptureData(
     capture_data->AddScopeStats(kScopeIds[i], std::move(stats));
```

Comment:
Can you explain me why we need both `AddScopeStats` and `UpdateScopeStats` ?

Reply:
AddScopeStats is only for testing purposes so you don't have to add 1000 different timers. I don't particularly like it and we should really update the test rather than have a test-only function, but it felt out of scope for this pr.

Reply:
Perfect, thanks for explaining. It would be nice if you add a TODO :)

Reply:
Done.